src/pages/chart/chart.tsx

```diff
@@ -128,12 +127,12 @@ const Chart = observer(({ show_digits_stats }: { show_digits_stats: boolean }) =
                     />
                 )}
                 chartType={chart_type}
-                isMobile={!isDesktop}
+                isMobile={isMobile}
                 enabledNavigationWidget={isDesktop}
                 granularity={granularity}
                 requestAPI={requestAPI}
-                requestForget={requestForget}
-                requestForgetStream={requestForgetStream}
+                requestForget={() => {}}
+                requestForgetStream={() => {}}
```

Comment:
Are these mandatory props? If we don't pass the function at all would it work?

Reply:
hey @shafin-deriv i tried removing i got some error so i kept it.